src/main.rs

```diff
@@ -746,6 +749,12 @@ fn get_setup(args: &[String]) -> Setup {
     if device == Some("?".into()) {
         backend(device, format);
         exit(0);
+    } else if device == Some("".into()) {
```

Comment:
Comparing against `Some()` seems unusual. How about:
```rust
if let Some(value) = device {
    // or use a match statement here:
    if value.is_empty() {
        //
    } else if value == "?" {
        //
    }
}
```

Reply:
I can do that.

Reply:
Done. I even took care of a similar case in parsing the password and username.